modules/document_repository/ajax/documentEditUpload.php

```diff
@@ -147,4 +184,26 @@
     }
 }
 
+/**
+ * Create a UUID v4 string.
+ *
+ * Source from comments:
+ * http://php.net/manual/en/function.com-create-guid.php
+ * Maybe move to Utilities class.
+ *
+ * @return string $version
+ */
+function uuid4()
```

Comment:
We're trying to include [return type declarations](https://secure.php.net/manual/en/functions.returning-values.php#functions.returning-values.type-declaration) for new code. 

Could you add `String` as a return type here?